pkg/logs/launchers/integration/launcher.go

```diff
@@ -170,6 +143,40 @@ func (s *Launcher) run() {
 	}
 }
 
+// receiveSources handles receiving incoming sources
+func (s *Launcher) receiveSources(cfg integrations.IntegrationConfig) {
+	sources, err := ad.CreateSources(cfg.Config)
+	if err != nil {
+		configError := fmt.Sprintf("Failed to create source for %s:", cfg.Config.Name)
+		ddLog.Error(configError, err)
+		return
+	}
+
+	for _, source := range sources {
+		// TODO: integrations should only be allowed to have one IntegrationType config.
+		if source.Config.Type == config.IntegrationType {
+			// This check avoids duplicating files that have already been created
+			// by scanInitialFiles
+			logFile, exists := s.integrationToFile[cfg.IntegrationID]
+
+			if !exists {
+				logFile, err = s.createFile(cfg.IntegrationID)
+				if err != nil {
+					sourceError := fmt.Sprintf("Failed to integration log file for %s:", source.Config.IntegrationName)
+					ddLog.Error(sourceError, err)
```

Comment:
```suggestion
					ddLog.Errorf("Failed to integration log file for %q: %v", source.Config.IntegrationName, err)
```

Also, failed to do what? If it is a file, would it make sense to also log the file path?